src/transactions/PathPaymentOpFrameBase.cpp

```diff
@@ -99,7 +99,8 @@ PathPaymentOpFrameBase::convert(AbstractLedgerTxn& ltx,
 
     switch (r)
     {
-    case ConvertResult::eFilterStop:
+    case ConvertResult::eFilterStopBadPrice:
```

Comment:
why handle `eFilterStopBadPrice` like this? I think it should just hit the `default` and crash as this should never happen